lib/LaTeXML/Core/Document.pm

```diff
@@ -884,7 +884,7 @@ sub closeToNode {
 
 # This closes all nodes until $node is closed.
 sub closeNode {
-  my ($self, $node) = @_;
+  my ($self, $node, $ifopen) = @_;
```

Comment:
so this is providing for a `maybeCloseNode` variant where `$ifopen` is the `$maybe_flag` ? :> Neat!